### ISSUE TITLE
The multiple Producer configuration is not working

### ISSUE THREAD

**Issue by sshemirani** [springwolf/springwolf-core#71]
Hello, it seems that the multiple producer support does not work, it was fixed on this issue [#59](https://github.com/springwolf/springwolf-core/issues/59)

Im attaching the project that does not work with multiple producers. 

[async-api-project.zip](https://github.com/springwolf/springwolf-core/files/8851387/async-api-project.zip)

Calling http://localhost:8080/springwolf/docs will bring the following json, but only one producer is in the json!

```
{
  "Async-api-test": {
    "asyncapi": "2.0.0",
    "info": {
      "title": "Async-api-test",
      "version": "0.0.1"
    },
    "servers": {
      "kafka": {
        "url": "localhost:9092",
        "protocol": "kafka"
      }
    },
    "channels": {
      "ACCOUNT": {
        "subscribe": {
          "bindings": {
            "kafka": {
              
            }
          },
          "message": {
            "name": "com.example.asyncapi.model.ClassB",
            "title": "ClassB",
            "payload": {
              "$ref": "#/components/schemas/ClassB"
            }
          }
        }
      }
    },
    "components": {
      "schemas": {
        "ClassA": {
          "type": "object"
        },
        "ClassB": {
          "type": "object"
        }
      }
    }
  }
}
```

Thanks

**Comment by stavshamir** [comment]
Oops, is this on the last release?

**Comment by sshemirani** [comment]
Yes indeed.

```
       <dependency>
            <groupId>io.github.springwolf</groupId>
            <artifactId>springwolf-kafka</artifactId>
            <version>0.5.0</version>
        </dependency>
```

**Comment by stavshamir** [comment]
Fixed, please use version `springwolf-kafka-0.6.1` and note the the `binding` field in `ProducerData` has changed to `operationBinding`. You can also use the new `KafkaProducerData` instead.

Thanks for providing the example project, it really helped me nail the problem.
